src/engine/world/world.cpp

```diff
@@ -1070,37 +1072,58 @@ void World::save(const std::filesystem::path& filename, bool isQuicksave)
   doc.save("meta", meta, meta);
   doc.save("data", *this, *this);
   doc.write();
+
+  if(metaPath.has_value())
+  {
+    serialization::YAMLDocument<false> metaCacheDoc{metaPath.value()};
+    metaCacheDoc.save("meta", meta, meta);
+    metaCacheDoc.write();
+  }
 }
 
 void World::save(const std::optional<size_t>& slot)
 {
   getPresenter().drawLoadingScreen(_("Saving..."));
   const auto filename = m_engine.getSavegamePath(slot);
-  save(filename, !slot.has_value());
+  const auto metaCachePath = m_engine.getSavegameMetaPath(slot);
+  save(filename, metaCachePath, !slot.has_value());
   getPresenter().disableScreenOverlay();
 }
 
 std::tuple<std::optional<SavegameInfo>, std::map<size_t, SavegameInfo>> World::getSavedGames() const
 {
-  auto getSavegameInfo = [](const std::filesystem::path& path) -> std::optional<SavegameInfo>
+  auto getSavegameInfo
+    = [](const std::filesystem::path& path, const std::filesystem::path& metaPath) -> std::optional<SavegameInfo>
```

Comment:
See other comment in `World::save`. The additional parameter then becomes obsolete.